### PR TITLE
Add greeting to README.md

### PR DESCRIPTION
**Pull request by tinkabel85**

### CODE REVIEW

**Comment by tinkabel85** [comment]
recovering pr

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and [git](https://github.com/Kottans/git-course) courses.
 You can try sending pull-requests to this repository proposing a change.
 The change can be anything you like, for example: fixing a typo in README.md. Consider this phrase, for example:
 
-*This was an intentionaly mispelld phrase*
+*This was an intentionaly misspelled phrase*
```
Comment (miroslav-kolomiets) [inline README.md:13]: Please translate this phrase and think about why it is here. And then return it back as it was ;)

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Good job! But please read comment and revert an intentionally misspelled phrase back.

File: README.md (hunk 1 above)
Comment (tinkabel85) [reply README.md]: fixed!

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hi! Seems like conflicts was not resolved correctly. Please check it.
Снимок экрана 2020-11-11 в 4 28 33 PM

**Comment by tinkabel85** [comment]
@miroslav-kolomiets Thank you, all fixed, now it's clean.

**Pull request merged**